enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProvider.java

```diff
@@ -251,8 +248,17 @@ private <T> T parseResponse(String body, Class<T> cls) throws PresenceCheckExcep
         }
     }
 
-    private String unescapeSlashes(String value) {
-        return value.replace("\\", "");
+    private static Image parseImage(final ClaimValidateResponse response) {
+        final String frameJpeg = unescapeSlashes(response.getFrame());
```

Comment:
I suggest to add a null check for `response`.

Reply:
Sure